### DISCUSSION TITLE
Rails compatibility: Don't apply aria-busy styles on <html> element

### DISCUSSION THREAD

**Discussion by mattbrictson** [picocss/pico#215] [Ideas]
Pico is great! I primary use it with Rails.

By default, [Rails will apply `aria-busy=true` on the `html` element](https://turbo.hotwired.dev/handbook/drive#displaying-progress) when browser navigation actions are in progress. This results in a visual glitch with Pico. Pico applies a loading spinner on any element with `aria-busy=true`, including the `html` element. So anytime you click a link or a button in a Rails app, Pico puts a spinner icon on the root of the page, causing all content on the page to shift down.

To fix this, it would be great if Pico could add a `:not(html)` to its `aria-busy=true` selector.

What do you think?

**Comment by mediafinger** [comment]
Hey @mattbrictson I just stumbled over the exact same issue, that @picocss together with @rails / @hotwired Turbo makes my page jump on every navigational click. Thx for explaining what is causing this issue.

Do you have any CSS styles to reset the `aria-busy=true` styling on `html`?  
I've managed to get rid of the animation, but my page is still jumping down on every click. :-/

**Reply by mattbrictson** [reply]
I haven't tested it thoroughly, but this snippet of CSS seems to work:

```css
html[aria-busy="true"]::before {
  all: unset;
}
```

**Reply by mediafinger** [reply]
That actually does the trick, thank you!

**Comment by mattbrictson** [comment]
This was fixed in #349!
